thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionResource.java

```diff
@@ -427,4 +423,54 @@ public Response detectionReplay(
 
     return Response.ok(result).build();
   }
+
+
+  /**
+   * Create a user-reported anomaly for a new detection pipeline
+   *
+   * @param detectionConfigId detection config id (must exist)
+   * @param startTime start time utc (in millis)
+   * @param endTime end time utc (in millis)
+   * @param metricUrn the metric urn of the anomaly
+   * @param feedbackType anomaly feedback type
+   * @param comment anomaly feedback comment (optional)
+   * @throws IllegalArgumentException if the anomaly function id cannot be found
+   * @throws IllegalArgumentException if the anomaly cannot be stored
+   */
+  @POST
+  @Path(value = "/report-anomaly/{detectionConfigId}")
+  @ApiOperation("Report a missing anomaly for a detection config")
+  public void createUserAnomaly(
```

Comment:
Maybe return a response with the created anomaly ID so that the caller knows the status of the api call.

Reply:
good idea. changed.